### DISCUSSION TITLE
Plugin Adoption

### DISCUSSION THREAD

**Discussion by markjacksonfishing** [jenkinsci/gitlab-plugin#1098] [General]
I am putting this plugin up for adoption.

**Comment by Stikus** [comment]
Thanks for info and pointing to this thread in #1087 
Here is another issue with discussion about it: #1027
According to this page: https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/#how-can-i-mark-a-plugin-for-adoption
you should add `adopt-this-plugin` to [topics](https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics#adding-topics-to-your-repository) (I think it is easier)

**Reply by omehegan** [reply]
@Stikus thanks, I have done that.
